verify.go

```diff
@@ -251,17 +285,6 @@ func verifyCertChain(ee *x509.Certificate, certs []*x509.Certificate, truststore
 	return
 }
 
-// MessageDigestMismatchError is returned when the signer data digest does not
```

Comment:
This error was moved to the top of the file alongside other errors for organization.